<!DOCTYPE HTML>
<!--
	Helios by HTML5 UP
	html5up.net | @n33co
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Integrated Microbiome Resource (IMR)</title>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="Information and access to services provided by the Dalhousie University Integrated Microbiome Resource" />
		<meta name="keywords" content="microbiome, Dalhousie, Halifax, NGS, 16S, 18S, metagenomics, metatranscriptomics, MiSeq, Illumina" />
		<!--[if lte IE 8]><script src="css/ie/html5shiv.js"></script><![endif]-->
		<script src="js/jquery.min.js"></script>
		<script src="js/jquery.dropotron.min.js"></script>
		<script src="js/jquery.scrolly.min.js"></script>
		<script src="js/jquery.onvisible.min.js"></script>
		<script src="js/skel.min.js"></script>
		<script src="js/skel-layers.min.js"></script>
		<script src="js/init.js"></script>
		<script>
			(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
			(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
			m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
			})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

			ga('create', 'UA-65099719-1', 'auto');
			ga('send', 'pageview');

		</script>

		<script>
			// Reduce the effect of FOUC (init.js always sets html display to block)
			var html_el = document.getElementsByTagName('html')[0];
			html_el.style.display = 'none';
		</script>

		<noscript>
			<link rel="stylesheet" href="css/skel.css" />
			<link rel="stylesheet" href="css/style.css" />
			<link rel="stylesheet" href="css/style-noscript.css" />
		</noscript>
		<!--[if lte IE 8]><link rel="stylesheet" href="css/ie/v8.css" /><![endif]-->
	</head>
	<body class="no-sidebar">

		<!-- Header -->
			<div id="header">

				<!-- Inner -->
					<div class="inner">
						<header>
							<h1><a href="index.html" id="logo">Integrated Microbiome Resource (IMR)</a></h1>
						</header>
					</div>
				
				<!-- Nav -->
					<nav id="nav">
						<ul>
							<li><a href="index.html">Home</a></li>
							<li><a href="about.html">About Us</a></li>
							<li><a href="achieve.html">Achievements</a></li>
							<li><a href="submissions.html">Submission Guidelines</a></li>
							<li><a href="queue.html">Real-Time Queue</a></li>
							<li><a href="pricing.html">Services/Pricing</a></li>
							<li><a href="protocols.html">Protocols</a></li>
							<li><a href="contact.html">Contact Us</a></li>
						</ul>
					</nav>

			</div>
			
		<!-- Main -->
			<div class="wrapper style1">

				<div class="container">
					<article id="main" class="special">
						<header>
							<h2>Protocols</h2>
						</header>
						<section>
							<p>The following protocol summaries are for the generation+sequencing of WGS/metagenome libraries (Illumina NextSeq or PacBio Sequel 2) or PCR amplicons with multiple barcodes (ie: "indices") on 
							either the Illumina MiSeq machine (paired-end mode) of length approx. 400-500 bp (300+300 bp with ~100-200 bp overlap) or of longer reads on the PacBio Sequel 2. It assumes an input 
							of up to 384 (380 samples + 4 PCR controls) for MiSeq or 192 (190 samples + 2 PCR controls) for Sequel 2. These protocols are a synthesis of multiple sources in the current scientific 
							literature and are summarized (primarily amplicon protocols) in the <b>"IMR paper"</b> here below that outlines <b>Microbiome Helper</b>, as well as on our <b>GitHub</b> site for 
							<b>Microbiome Helper</b> which outlines all the steps to follow for both the bioinformatics processing and the step-by-step wet-lab protocols (soon to be converted to Protocols.io):
							<ul class="default">
								<li><a href="https://journals.asm.org/doi/10.1128/mSystems.00127-16" target="new"><b>Comeau AM, Douglas GM, Langille M.</b> 2017. Microbiome Helper: A custom and streamlined workflow for microbiome research. <i>mSystems</i>, 2:e00127-16.</a>
								<li><a href="https://github.com/LangilleLab/microbiome_helper/wiki" target="new"><b>Microbiome Helper GitHub</b> - Homepage with access to all bioinformatics pipelines</a>
								<li><a href="https://github.com/LangilleLab/microbiome_helper/wiki/Microbiome-Amplicon-Sequencing-Workflow" target="new"><b>Microbiome Helper GitHub (Wet-lab)</b> - Specific page for the wet-lab protocols</a>
							</ul>

						</section>
						<section>
							<header>
								<a name="collect"></a><h3>Sample Collection</h3>
							</header>
							<p>Sample collection issues can be very specific to your sample type (ie: the medium which you are sampling) - many of our IMR projects so far focus on water/soil "environmental" 
							samples or fecal matter. For the former, varying volumes of water are typically collected onto filters or a few grams of soil are collected into tubes and frozen at -20&deg;C or 
							-80&deg;C, preferrably dry without a storage buffers. For the latter, fresh fecal pellets are collected from mice or human stool is sampled and then frozen immediately at -20&deg;C 
							or -80&deg;C (again without buffer). All samples are kept frozen until extraction below. For transcriptomics approaches for environmental, lab culture or biopsy samples (not 
							recommended for stool), samples must be immediately flash-frozen in liquid nitrogen after collection.
							</p>
						</section>
						<section>
							<header>
								<a name="extract"></a><h3>DNA/RNA Extraction</h3>
							</header>
							<p>DNA/RNA is extracted from the samples using the method/kit appropriate to the specific samples - this may also be a choice you make for your personal samples since you have 
							experience with them, or you may wish to follow our choice extraction methods. There is no general consensus in the literature on choice of kits, other than to say all of them 
							affect the final profiles to some degree and that bead-beating is most probably a must for difficult materials (and/or with Gram+ves and protist cysts) - we have currently evaluated 
							and are using the <b>QIAGEN PowerFecal DNA Kit</b> with mouse pellets and human fecal samples; the <b>QIAGEN PowerSoil DNA Kit</b> for soil particles; and the <b>QIAGEN PowerWater 
							DNA Kit</b> for water filters. We tend to also use the <b>QIAGEN PowerFecal DNA Kit</b> as a general kit for swab and saliva extractions, as it has good inhibitor removal+overall 
							performance, as well as the necessary bead-beating (note that swab samples are notoriously difficult/variable in output due to low biomass). We have also tested various DNA kits 
							for use with human urine without much success due to their low target biomass. We have not yet ventured into RNA kits for metatranscriptomes, but this is something that we will be 
							examining in the near future. Quantification and quality-checks are done (via PicoGreen/Qubit [primarily] and NanoDrop) to verify success. <i>Optional: A gel can be run to verify 
							integrity (generally unnecessary for PCR-only studies, but required for WGS sequencing).</i>
							</p>
						</section>
						<section>
							<header>
								<a name="library"></a><h3>Library Preparation</h3>
							</header>
						</section>
						<section>
							<header>
								<a name="libAmp"></a><h3><i>&rarr; 16S/18S/ITS Amplicons</i></h3>
							</header>
							<p>Amplicon fragments are PCR-amplified from the DNA in duplicate using separate template dilutions (generally 1:1 & 1:10) using the high-fidelity <b>Phusion+</b> polymerase. A 
							single round of PCR is done using "fusion primers" (Illumina adaptors + indices + specific regions) targeting various sub-regions of the <b>16S</b> (Bacteria/Archaea), <b>18S</b> 
							(Eukarya, incl. Fungi) or <b>ITS2</b> (Fungi only) genes with multiplexing which allows up to 380 samples to be run. Preparation for PacBio Sequel 2 is essentially the same, except 
							<b>full-length 16S/18S/ITS</b> fusion primers (PacBio adaptors + barcodes + specific regions) are used instead. PCR products are verified visually by running on a high-throughput 
							<b>Hamilton Nimbus Select</b> robot using <b>Coastal Genomics Analytical Gels</b>. The PCR reactions from the same samples are pooled in one plate, then cleaned-up and normalized 
							using the high-throughput <b>Charm Biotech Just-a-Plate 96-well Normalization Kit</b>. Up to 380 (Illumina) or 190 (PacBio) samples are then pooled to make one library which is then 
							quantified fluorometrically before sequencing.
							</p>
						</section>
						<section>
							<center><h3><center><i>Currently Available Amplicon Targets/Primers (recommended sets in bold)</i></center></h3>
							<table class="primers">
								<tr><th colspan=5>																																												<th colspan=6>Primer Set Coverages (SILVA TestPrime, 0-2 mismatches)<sup>a</sup>
								<tr><th align=left>Primer Targets<th>Region(s)		<th>Forward Primer			<th>Reverse Primer			<th>Source(s)																												<th>Archaea					<th>Bacteria					<th>Cyanos					<th>Eukarya					<th>mtDNA					<th>chlDNA
								<tr><th colspan=11 style="background:#eeeeee">Illumina MiSeq short variable region targets
								<tr><td><b>Universal		<td><b>V4-V5<sup>b</sup><td>515FB = GTGYCAGCMGCCGCGGTAA		<td>926R = CCGYCAATTYMTTTRAGTTT		<td><a href="https://sfamjournals.onlinelibrary.wiley.com/doi/full/10.1111/1462-2920.13023" target="new">Parada 2015</a> / <a href="http://msystems.asm.org/content/1/1/e00009-15" target="new">Walters 2015</a>			<td style="background-color:palegreen">81-93%	<td style="background-color:palegreen">85-95%	<td style="background-color:palegreen">85-94%	<td style="background-color:palegreen">81-94%	<td style="background-color:yellow">57-77%	<td style="background-color:palegreen">81-93%
								<tr><td><b>Archaea-specific	<td><b>V6-V8		<td>A956F = TYAATYGGANTCAACRCC		<td>A1401R = CRGTGWGTRCAAGGRGCA		<td><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0027492" target="new">Comeau 2011</a>															<td style="background-color:palegreen">71-82%	<td>-						<td>-						<td style="background-color:yellow">0-89%	<td style="background-color:#ffcccc">0-1%	<td style="background-color:#ffcccc">0-1%
								<tr><td><b>Bacteria-specific	<td><b>V6-V8		<td>B969F = ACGCGHNRAACCTTACC		<td>BA1406R = ACGGGCRGTGWGTRCAA		<td><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0027492" target="new">Comeau 2011</a>															<td style="background-color:#ffcccc">0-14%	<td style="background-color:palegreen">72-83%	<td style="background-color:yellow">66-88%	<td style="background-color:#ffcccc">0-1%	<td style="background-color:yellow">14-75%	<td style="background-color:yellow">47-87%
								<tr><td><b>Eukaryote-specific	<td><b>V4		<td>E572F = CYGCGGTAATTCCAGCTC		<td>E1009R = AYGGTATCTRATCRTCTTYG	<td><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0027492" target="new">Comeau 2011</a>															<td>-						<td>-						<td>-						<td style="background-color:palegreen">54-92%	<td style="background-color:#ffcccc">1%		<td style="background-color:#ffcccc">1%
								<tr><td><b>Fungi-specific	<td><b>ITS2<sup>c</sup>	<td>ITS86(F) = GTGAATCATCGAATCTTTGAA	<td>ITS4(R) = TCCTCCGCTTATTGATATGC	<td><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0097629" target="new">Op De Beeck 2014</a>														<td>n/a						<td>n/a						<td>n/a						<td>n/a						<td>n/a						<td>n/a
								<tr><td>Bacteria-specific	<td>V1-V3<sup>d</sup>	<td>27Fmod = AGRGTTTGATCMTGGCTCAG	<td>519R = GWATTACCGCGGCKGCTG		<td><a href="https://academic.oup.com/dnaresearch/article/20/3/241/368334" target="new">Kim 2013</a> / <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC391288/" target="new">Lane 1985</a>					<td>-						<td style="background-color:palegreen">73-93%	<td style="background-color:yellow">44-89%	<td>-						<td style="background-color:yellow">9-75%	<td style="background-color:yellow">24-87%
								<tr><td>Bacteria-specific ("Illumina")	<td>V3-V4	<td>341F = CCTACGGGNGGCWGCAG		<td>805R = GACTACHVGGGTATCTAATCC	<td>Illumina / <a href="https://academic.oup.com/nar/article/41/1/e1/1164457" target="new">Klindworth 2013</a>																<td style="background-color:yellow">0-90%	<td style="background-color:palegreen">83-95%	<td style="background-color:palegreen">71-93%	<td>-						<td style="background-color:yellow">11-54%	<td style="background-color:yellow">49-90%
								<tr><td>Bacteria+Archaea-specific ("EMP")<td>V4<sup>e</sup><td>515FB = GTGYCAGCMGCCGCGGTAA	<td>806RB = GGACTACNVGGGTWTCTAAT	<td><a href="http://msystems.asm.org/content/1/1/e00009-15" target="new">Walters 2015</a>																		<td style="background-color:palegreen">84-96%	<td style="background-color:palegreen">84-95%	<td style="background-color:palegreen">76-92%	<td style="background-color:#ffcccc">0-19%	<td style="background-color:yellow">52-89%	<td style="background-color:palegreen">64-89%
								<tr><td>Cyano-specific		<td>V3-V4<sup>f</sup>	<td>CYA359F = GGGGAATYTTCCGCAATGGG	<td>CYA781R = GACTACWGGGGTATCTAATCCCWTT	<td><a href="https://aem.asm.org/content/63/8/3327" target="new">N&uuml;bel 1997</a>																			<td>-						<td style="background-color:#ffcccc">2-5%	<td style="background-color:yellow">58-88%	<td>-						<td style="background-color:#ffcccc">1-3%	<td style="background-color:yellow">35-79%
								<tr><th colspan=11 style="background:#eeeeee">PacBio Sequel entire region targets
								<tr><td><b>Bacteria-specific	<td><b>Full 16S		<td>27F(Paliy) = AGRGTTYGATYMTGGCTCAG	<td>1492R = RGYTACCTTGTTACGACTT		<td><a href="https://aem.asm.org/content/75/11/3572.long" target="new">Paliy 2009</a> / Lane 1991																	<td>-						<td style="background-color:palegreen">72-83%	<td style="background-color:palegreen">72-87%	<td>-						<td style="background-color:yellow">42-74%	<td style="background-color:palegreen">65-84%
								<tr><td><b>Eukaryote-specific	<td><b>Full 18S<sup>g</sup><td>NSF4/18 = CTGGTTGATYCTGCCAGT	<td>EukR = TGATCCTTCTGCAGGTTCACCTAC	<td><a href="https://www.sciencedirect.com/science/article/abs/pii/S0723202089800669" target="new">Hendriks 1989</a> / <a href="https://www.sciencedirect.com/science/article/abs/pii/0378111988900662" target="new">Medlin 1988</a>	<td style="background-color:#ffcccc">0-14%	<td>-						<td>-						<td style="background-color:palegreen">82-92%	<td style="background-color:#ffcccc">2%		<td style="background-color:#ffcccc">1%
								<tr><td><b>Fungi-specific	<td><b>Full ITS		<td>ITS1FKYO2 = TAGAGGAAGTAAAAGTCGTAA	<td>ITS4KYO1 = TCCTCCGCTTWTTGWTWTGC	<td><a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0040863" target="new">Toju 2012</a>															<td>n/a						<td>n/a						<td>n/a						<td>n/a						<td>n/a						<td>n/a
							</table></center>
							<b><i>Notes/Details:</i></b>
							<br>a) Green = good coverage; Yellow = moderate coverage; Red = poor/trace coverage. Chloroplast DNA coverage applies to Cyano-derived chloroplasts only.
							<br>b) The V4-V5 primer set is our overall recommended set, but should not be used for bacterial diversity in samples with substantial eukaryote "host/associated" contamination (use V6-V8 sets instead, however these are still susceptible to Euk mito contamination).
							<br>c) Although both ITS1 and ITS2 primers have substantial Ascomycete or Basidiomycete biases, the ITS2 region seems to be the more recommended region of the two with highest coverage of other phyla.
							<br>d) The 519R primer with this sequence is sometimes called "mod" or "modbio", but the stated sequence here is the actual Lane original sequence.
							<br>e) Note this is the newer EMP set, the older/original primers being 515F (GTGCCAGCMGCCGCGGTAA) + 806R (GGACTACHVGGGTWTCTAAT). These primers are not recommended for 2x300bp sequencing (which we always perform) is its size is too small - use V4V5 instead.
							<br>f) The Cyano primers in our hands tend to perform less well in PCR+sequencing than the other sets listed here, but are the only alternative for more Cyano-specific studies. However, note that they will not give more specific annotations than the other bacterial amplicons.
							<br>g) The listed coverage for this primer set is from only the NSF4/18 forward primer values as there is insufficient coverage of the EukR region in SILVA to get reliable results.
						</section>
						<section>
							<header>
								<a name="libMetaG"></a><h3><i>&rarr; (Meta)genomes ("Shotgun")</i></h3>
							</header>
							<p>Microbial (or mtDNA) genomes and community metagenomes are prepared either using: 1) the <b>Illumina Nextera Flex</b> kit for MiSeq+NextSeq, which requires a very small amount 
							of starting material (1 ng) as it is a PCR-based library preparation procedure; or 2) the <b>PacBio SMRTbell Express Template Prep</b> kit for the Sequel, which requires more HMW DNA 
							since it is not PCR-based. For the Illumina method, samples are "tagmented" (enzymatically "sheared" and tagged with adaptors), PCR amplified while adding barcodes, purified using 
							columns or beads, normalized using Illumina beads or manually, then pooled for loading onto the MiSeq or NextSeq. For the PacBio method, the HMW DNA is mechanically 
							sheared (<b>Covaris gTubes</b>), optionally size-selected, repaired, converted into SMRTbell libraries (covalently closed circles), cleaned-up and normalized, then pooled for 
							loading onto the Sequel. 
							</p>
						</section>
						<section>
							<header>
								<a name="libMetaT"></a><h3><i>&rarr; (Meta)transcriptomes ("RNA-Seq")</i></h3>
							</header>
							<p>We recently completed our evaluation of multiple RNA-Seq kits with rRNA depletion and have selected the <b>QIAseq FastSelect depletion + Stranded Total RNA</b> kits for the	
							production of sufficiently rRNA-depleted libraries (using three rRNA modules = Bacteria + Plant[covers Protists] + Fungi). Briefly, pure RNA samples are rRNA depleted, then the 
							remaining mRNAs are converted to cDNA in a way that maintains stranded information, tagged with Illumina adaptors+barcodes, PCR amplified, purified using beads, then normalized for 
							pooling + loading onto the NextSeq.
							</p>
						</section>
						<section>
							<header>
								<a name="NGS"></a><h3>Next-Generation Sequencing</h3>
							</header>
							<p>
								Amplicon samples and small genomes are run on our <b>Illumina MiSeq</b> using 300+300 bp paired-end V3 chemistry which allows for overlap and stitching togther of paired 
								amplicon reads into one full-length read of higher quality. Output is generally ~20-22 million raw reads and ~13 Gb of sequence = ~50,000 reads per sample for 380 amplicons. 
								Long amplicons and <i>de novo</i> genomes are run on our <b>PacBio Sequel 2</b> using new 8M chips which output roughly 240-320 Gb of sequence per cell of variable (long) 
								lengths. For larger metagenomic/metatranscriptomic projects, we run on our <b>Illumina NextSeq 550</b> using 150+150 bp paired-end "high output" chemistry generating up to 
								~400 million raw reads and ~120 Gb of sequence.
							</p>
						</section>
						<section>
							<header>
								<a name="bioinfo"></a><h3>Bioinformatics Analyses</h3>
							</header>
							<p>
								Details of our amplicon and metagenomics pipelines are available at <a href="https://github.com/mlangill/microbiome_helper/wiki" target="new">https://github.com/mlangill/microbiome_helper/wiki</a>, 
								but the following is a summary of the major deliverables clients will receive (analyses will require a "mapping file" from the clients containing any relevant metadata for 
								the study):
								<p><i><b>16S/18S/ITS Amplicon Analysis</b></i>
								<ul class="default">
									<li>Final ASV tables in text, BIOM and STAMP formats
									<li>Accompanying QIIME2-formatted mapping/metadata file
									<li>FASTA file of representative sequences (one per ASV)
									<li>Phylogenetic tree of ASVs placed within reference sequences
									<li>Taxonomic assignment files at various levels (ex: phylum, genus, etc.)
									<li>Alpha-diversity rarefaction plots + statistics
									<li>Beta-diversity UniFrac plots
									<li>Logfiles from the various major steps in the QC process
									<li>Functional prediction files generated from PICRUSt2.0 (if requested)
								</ul>
								<p><i><b>Metagenomics/Metatranscriptomics Analysis</b></i>
								<ul class="default">
									<li>FASTA files of the final sequences screened to remove human (or other) contaminants (available upon request)
									<li>Taxonomic composition of the samples from Kraken2 (text and STAMP files)
									<li>Stratified (by taxa) and unstratified functional prediction files generated from MMseqs2+Kraken2 (text and STAMP files) for individual UniRef90 gene families and MetaCyc pathways
								</ul>
								<p><b>Custom Bioinformatics:</b> Additional bioinformatic analyses can be requested at an hourly rate or through research collaboration. Please contact us for more details.
							</p>
						</section>
					</article>
					</div>
				</div>

			</div>

		<!-- Footer -->
			<div id="footer">
							<!-- Copyright -->
								<div class="copyright">
									<ul class="menu">
										<li>&copy; IMR (2014+). All rights reserved.</li><li>Design template: <a href="http://html5up.net" target="new">HTML5 UP</a></li>
									</ul>
								</div>
							
			</div>

	</body>
</html>
