<!DOCTYPE HTML>
<!--
	Helios by HTML5 UP
	html5up.net | @n33co
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>CGEB - Integrated Microbiome Resource (IMR)</title>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="Information and access to services provided by the Dalhousie University Integrated Microbiome Resource" />
		<meta name="keywords" content="microbiome, Dalhousie, Halifax, NGS, 16S, 18S, metagenomics, metatranscriptomics, MiSeq, Illumina" />
		<!--[if lte IE 8]><script src="css/ie/html5shiv.js"></script><![endif]-->
		<script src="js/jquery.min.js"></script>
		<script src="js/jquery.dropotron.min.js"></script>
		<script src="js/jquery.scrolly.min.js"></script>
		<script src="js/jquery.onvisible.min.js"></script>
		<script src="js/skel.min.js"></script>
		<script src="js/skel-layers.min.js"></script>
		<script src="js/init.js"></script>
		<noscript>
			<link rel="stylesheet" href="css/skel.css" />
			<link rel="stylesheet" href="css/style.css" />
			<link rel="stylesheet" href="css/style-desktop.css" />
			<link rel="stylesheet" href="css/style-noscript.css" />
		</noscript>
		<!--[if lte IE 8]><link rel="stylesheet" href="css/ie/v8.css" /><![endif]-->
	</head>
	<body class="no-sidebar">

		<!-- Header -->
			<div id="header">

				<!-- Inner -->
					<div class="inner">
						<header>
							<h1><a href="index.html" id="logo">Integrated Microbiome Resource (IMR)</a></h1>
						</header>
					</div>
				
				<!-- Nav -->
					<nav id="nav">
						<ul>
							<li><a href="index.html">Home</a></li>
							<li><a href="about.html">About Us</a></li>
							<li><a href="achieve.html">Achievements</a></li>
							<li><a href="submissions.html">Project/Sample Submissions</a></li>
							<li><a href="pricing.html">Pricing</a></li>
							<li>
								<a href="protocols.html">Protocols</a>
								<ul>
									<li><a href="protocols.html#collect">Sample Collection</a></li>
									<li><a href="protocols.html#extract">DNA/RNA Extraction</a></li>
									<li>
										<a href="protocols.html#library">Library Preparation &hellip;</a>
										<ul>
											<li><a href="protocols.html#lib16S">16S Amplicons</a></li>
											<li><a href="protocols.html#lib18S">18S Amplicons</a></li>
											<li><a href="protocols.html#libMetaG">(Meta)genomes</a></li>
											<li><a href="protocols.html#libMetaT">(Meta)transcriptomes</a></li>
										</ul>
									</li>
									<li><a href="protocols.html#NGS">NGS Sequencing</a></li>
									<li><a href="protocols.html#bioinfo">Bioinformatics</a></li>
								</ul>
							</li>
							<li><a href="contact.html">Contact Us</a></li>
						</ul>
					</nav>

			</div>
			
		<!-- Main -->
			<div class="wrapper style1">

				<div class="container">
					<article id="main" class="special">
						<header>
							<h2>Protocols</h2>
						</header>
						<section>
							<p>The following protocol summaries are for the generation of paired-end sequencing reads of 16S or 18S PCR amplicons with multiple barcodes (ie: "indices") on the Illumina MiSeq 
							machine of length approx. 400-500 bp (300+300 bp with ~100 bp overlap). It assumes an input of up to 384 (380 samples + 4 PCR controls). These protocols are a synthesis of multiple 
							sources in the current scientific literature, but draws heavily upon the following four sources:
							<ul class="default">
								<li><b>Comeau AM, Li WK, Tremblay JE, Carmack EC, Lovejoy C.</b> 2011. Arctic Ocean microbial community structure before and after the 2007 record sea ice minimum. <i>PLoS ONE</i>, 6:e27492. PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/22096583" target="new">22096583</a>. [<font style="color: red;">primer design/sequences and PCR set-up</font>]
								<li><b>Fadrosh DW, Ma B, Gajer P, Sengamalay N, Ott S, Brotman RM, Ravel J.</b> 2014. An improved dual-indexing approach for multiplexed 16S rRNA gene sequencing on the Illumina MiSeq platform. <i>Microbiome</i>, 2:6. PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/24558975" target="new">24558975</a>. [<font style="color: red;">Illumina indexing set-up and post-PCR treatment</font>]
								<li><b>Earth Microbiome Project (EMP)</b> at <a href="http://www.earthmicrobiome.org/emp-standard-protocols/" target="new">www.earthmicrobiome.org/emp-standard-protocols/</a> [<font style="color: red;">blocking protocol for eukaryote contaminants</font>]
								<li><b>Human Microbiome Project (HMP)</b> at <a href="http://hmpdacc.org/resources/tools_protocols.php" target="new">hmpdacc.org/resources/tools_protocols.php</a> [<font style="color: red;">general considerations/extraction for stool samples</font>]
							</ul>

						</section>
						<section>
							<header>
								<a name="collect"></a><h3>Sample Collection</h3>
							</header>
							<p>Sample collection issues can be very specific to your sample type (ie: the medium which you are sampling) - many of our IMR projects so far focus on water column "environmental" 
							samples or fecal matter. For the former, varying volumes of water are typically collected onto filters and frozen at -20&deg;C or -80&deg;C in a storage buffer. For the latter, fresh fecal 
							pellets are collected from mice or human stool is sampled and then frozen immediately at -20&deg;C or -80&deg;C (without buffer). All samples are kept frozen until extraction below.
							</p>
						</section>
						<section>
							<header>
								<a name="extract"></a><h3>DNA/RNA Extraction</h3>
							</header>
							<p>DNA/RNA is extracted from the samples using the method/kit appropriate to the specific samples - this may also be a choice you make for your personal samples since you have experience 
							with them, or you may wish to follow our choice extraction methods. There is no general consensus in the literature on choice of kits, other than to say all of them affect the 
							final profiles to some degree and that bead-beating is most probably a must for difficult materials (and/or with much Gram+ves and protist cysts) - we have currently evaluated and
							are using the <b>MO BIO PowerFecal DNA Kit</b> with mouse pellets and human fecal samples. We are also testing various DNA kits for use with human urine. We have not yet ventured into 
							RNA kits for metatranscriptomes, but this is something that we will be examining in the near future. Quantification and quality-checks are done (via NanoDrop or PicoGreen/Qubit) to 
							verify success. <i>Optional: A gel can be run to verify integrity (generally unnecessary for PCR-only studies, but required for shotgun metagenomic sequencing).</i>
							</p>
						</section>
						<section>
							<header>
								<a name="library"></a><h3>Library Preparation</h3>
							</header>
						</section>
						<section>
							<header>
								<a name="lib16S"></a><h3><i>&rarr; 16S Amplicons</i></h3>
							</header>
							<p>16S fragments are PCR-amplified from the DNA in duplicate or triplicate using separate template dilutions (generally 1:1, 1:2, 1:10) using the high-fidelity <b>Phusion</b> 
							polymerase. A single round of PCR is done using "fusion primers" (Illumina adaptors + indices + specific regions) targeting the V6/V7/V8 region (438 bp usable sequence) with 
							multiplexing which allows up to 384 samples to be run. PCR products are verified visually by running a high-throughput <b>Invitrogen 96-well E-gel</b>. Any samples with 
							failed PCRs (or spurious bands) are re-amplified by optimizing PCR conditions to produce correct bands in order to complete the sample plate before continuing. The PCR reactions 
							from the same samples are pooled in one plate, then cleaned-up and normalized using the high-throughput <b>Invitrogen SequalPrep 96-well Plate Kit</b>. The (up to) 384 samples are 
							then pooled to make one library which is then quantified and quality-checked as above before sequencing.
							</p>
						</section>
						<section>
							<header>
								<a name="lib18S"></a><h3><i>&rarr; 18S Amplicons</i></h3>
							</header>
							<p>18S fragments are PCR-amplified from the DNA in duplicate or triplicate using separate template dilutions (generally 1:1, 1:2, 1:10) using the high-fidelity <b>Phusion</b> 
							polymerase. A single round of PCR is done using "fusion primers" (Illumina adaptors + indices + specific regions) targeting the V4 region (variable size, avg. 580 bp) with 
							multiplexing which allows up to 384 samples to be run. In cases where human or mouse 18S DNA contamination will be of great concern, a <b>mammalian blocking primer</b> is 
							used following the protocol from the EMP listed above. PCR products are verified visually by running a high-throughput <b>Invitrogen 96-well E-gel</b>. Any samples with 
							failed PCRs (or spurious bands) are re-amplified by optimizing PCR conditions to produce correct bands in order to complete the sample plate before continuing. The PCR	reactions 
							from the same samples are pooled in one plate, then cleaned-up and normalized using the high-throughput <b>Invitrogen SequalPrep 96-well Plate Kit</b>. The (up to) 384 samples are 
							then pooled to make one library which is then quantified and quality-checked as above before sequencing.
							</p>
						</section>
						<section>
							<header>
								<a name="libMetaG"></a><h3><i>&rarr; (Meta)genomes ("Shotgun")</i></h3>
							</header>
							<p>Microbial (or mtDNA) genomes and community metagenomes are prepared using the <b>Illumina Nextera XT</b> kit which requires a very small amount of starting material 
							(1 ng) as it is a PCR-based library preparation procedure. Briefly, samples are "tagmented" (enzymatically "sheared" and tagged with adaptors), PCR amplified while adding barcodes, 
							purified using AMPure beads, normalized using Illumina beads, then pooled for loading onto the MiSeq.
							</p>
						</section>
						<section>
							<header>
								<a name="libMetaT"></a><h3><i>&rarr; (Meta)transcriptomes ("RNA-Seq")</i></h3>
							</header>
							<p>The choice of library prep for metatranscriptomes varies somewhat, depending a few factors: 1) source organism; 2) anticipated/measured RNA quality/quantity; 3) whether 
							"strandedness" is a concern; and 4) whether mRNA-only, whole transcriptomes and/or small RNAs wish to be analyzed. The base kit for these appplications is <b>Illumina TruSeq</b> - 
							see the	<a href="http://www.illumina.com/applications/sequencing/rna.html" target="new">Illumina RNA Sequencing</a> application pages for further descriptions of the kit variants 
							for the different applications and access to an interactive library prep selector (bottom-right of the page).
							</p>
						</section>
						<section>
							<header>
								<a name="NGS"></a><h3>Next-Generation Sequencing</h3>
							</header>
							<p>
								Amplicon samples, small metagenomic sets, and genomes are run in-house on our <b>Illumina MiSeq</b>, most often using 300+300 bp paired-end V3 chemistry which 
								allows for overlap and stitching togther of paired amplicon reads into one full-length read of higher quality. Output is generally ~25 million reads and ~15 Gb of sequence = 
								~80,000 reads per sample for 384 16S/18S amplicons. For larger metagenomic/metatranscriptomic projects, sequencing is currently outsourced to locations that have an 
								<b>Illumina HiSeq</b>, such as the Genome-Qu&eacute;bec/McGill Innovation Centre. However, funding for an <b>Illumina NextSeq 500</b> has been recently secured and, once 
								installed (Spring 2016), will provide us the capacity to conduct medium-to-large projects locally.
							</p>
						</section>
						<section>
							<header>
								<a name="NGS"></a><h3>Bioinformatics Analyses</h3>
							</header>
							<p>
								Details of our 16S and metagenomics pipelines are available at <a href="https://github.com/mlangill/microbiome_helper" target="new">https://github.com/mlangill/microbiome_helper</a>, 
								but the following is a summary of the major deliverables clients will receive (analyses will require a "mapping file" from the clients containing any relevant metadata for 
								the study):
								<p><i><b>16S/18S Amplicon Analysis</b></i>
								<ul class="default">
									<li>Combined FASTA file of the quality-controlled sequences (formatted for use in QIIME)
									<li>Final OTU tables in text, BIOM and STAMP formats (from open-ref. picking at 97%)
									<li>FASTA file of representative sequences (one per OTU)
									<li>Taxonomic assignment files at various levels (ex: phylum, genus, etc.)
									<li>Alpha-diversity rarefaction plots
									<li>Beta-diversity UniFrac plots
									<li>Functional prediction files generated from PICRUSt
								</ul>
								<p><i><b>Metagenomics Analysis</b></i>
								<ul class="default">
									<li>FASTA files of the final sequences screened to remove human (or other) contaminants
									<li>Taxonomic composition of the samples from MetaPhlAn 2.0 (text and STAMP files)
									<li>Functional prediction files generated from HUMAnN (text and STAMP files) for individual KO numbers, KEGG modules and KEGG pathways
								</ul>
								<p><b>Custom Bioinformatics:</b> Additional bioinformatic analyses can be requested at an hourly rate or through research collaboration. Please contact us for more details.
							</p>
						</section>
					</article>
					</div>
				</div>

			</div>

		<!-- Footer -->
			<div id="footer">
							<!-- Copyright -->
								<div class="copyright">
									<ul class="menu">
										<li>&copy; CGEB-IMR (2014+). All rights reserved.</li><li>Design template: <a href="http://html5up.net" target="new">HTML5 UP</a></li>
									</ul>
								</div>
							
			</div>

	</body>
</html>