<!DOCTYPE HTML>
<!--
	Helios by HTML5 UP
	html5up.net | @n33co
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>CGEB - Integrated Microbiome Resource (IMR)</title>
		<meta http-equiv="content-type" content="text/html; charset=utf-8" />
		<meta name="description" content="Information and access to services provided by the Dalhousie University Integrated Microbiome Resource" />
		<meta name="keywords" content="microbiome, Dalhousie, Halifax, NGS, 16S, 18S, metagenomics, metatranscriptomics, MiSeq, Illumina" />
		<!--[if lte IE 8]><script src="css/ie/html5shiv.js"></script><![endif]-->
		<script src="js/jquery.min.js"></script>
		<script src="js/jquery.dropotron.min.js"></script>
		<script src="js/jquery.scrolly.min.js"></script>
		<script src="js/jquery.onvisible.min.js"></script>
		<script src="js/skel.min.js"></script>
		<script src="js/skel-layers.min.js"></script>
		<script src="js/init.js"></script>
<script>
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-65099719-1', 'auto');
  ga('send', 'pageview');

</script>
		<noscript>
			<link rel="stylesheet" href="css/skel.css" />
			<link rel="stylesheet" href="css/style.css" />
			<link rel="stylesheet" href="css/style-desktop.css" />
			<link rel="stylesheet" href="css/style-noscript.css" />
		</noscript>
		<!--[if lte IE 8]><link rel="stylesheet" href="css/ie/v8.css" /><![endif]-->
	</head>
	<body class="right-sidebar">

		<!-- Header -->
			<div id="header">

				<!-- Inner -->
					<div class="inner">
						<header>
							<h1>Integrated Microbiome Resource (IMR)</h1>
						</header>
					</div>
				
				<!-- Nav -->
					<nav id="nav">
						<ul>
							<li><a href="index.html">Home</a></li>
							<li><a href="about.html">About Us</a></li>
							<li><a href="achieve.html">Achievements</a></li>
							<li><a href="submissions.html">Project/Sample Submissions</a></li>
							<li><a href="pricing.html">Pricing</a></li>
							<li>
								<a href="protocols.html">Protocols</a>
								<ul>
									<li><a href="protocols.html#collect">Sample Collection</a></li>
									<li><a href="protocols.html#extract">DNA/RNA Extraction</a></li>
									<li>
										<a href="protocols.html#library">Library Preparation &hellip;</a>
										<ul>
											<li><a href="protocols.html#lib16S">16S Amplicons</a></li>
											<li><a href="protocols.html#lib18S">18S Amplicons</a></li>
											<li><a href="protocols.html#libMetaG">(Meta)genomes</a></li>
											<li><a href="protocols.html#libMetaT">(Meta)transcriptomes</a></li>
										</ul>
									</li>
									<li><a href="protocols.html#NGS">NGS Sequencing</a></li>
									<li><a href="protocols.html#bioinfo">Bioinformatics</a></li>
								</ul>
							</li>
							<li><a href="contact.html">Contact Us</a></li>
						</ul>
					</nav>

			</div>
			
		<!-- Main -->
			<div class="wrapper style1">

				<div class="container">
					<div class="row 200%">
						<div class="4u" id="content">
							<article id="main">
								<header>
									<h2>Project/Sample Submissions</h2>
								</header>
								<section>
									<p>
										Contact the <a href="mailto:andre.comeau@dal.ca"><b>IMR Manager</b></a> directly to initiate your project's consultation, then 
										download our Excel form:
										<p><b><a href="docs/IMR-AmpliconSubmissionSheet_v6(your_last_name_here).xlsx" target="new">Amplicon Sample Submission Form <img src="images/excel-icon.png" height="30" style="vertical-align:middle"></a></b>
										<p>This form is to be filled out with all of your details for tracking purposes and has sample preparation and shipping guidelines 
										(<b><i><font style="color: red;">please pay careful attention for international clients</font></b></i>). You will be given a unique project/sample identifier in the 
										form of "<b>IMR-YYYYMM-00</b>" which will allow sample tracking, will show up on your billing information and be listed in the queue 
										at right. 
									</p>
								</section>
								<section>
									<header>
										<h3>Sample queuing and real-time status</h3>
									</header>
									<p>
										In order to offer as flexible (and economical) a service as possible to the community, we will not be insisting that clients 
										fulfill a complete multiplexed run of 380 (380 + 4 negatives = 384) samples. We offer the option of sequencing less samples, but 
										with the caveat that your samples may take a bit longer in the pipeline as we wait for the remainder of the 380 slots to be 
										filled by other projects/individuals. Of course, you will always have the option of expediting your service by purchasing the 
										complete run at any time (even with less than 380 samples).
									</p>
									<p>
										We will attempt to maintain as close to real-time as possible project/sample information on the queue display on the right 
										of this page. This will allow you to monitor the progress of your samples through the IMR workflow. It will also allow 
										prospective clients to get a sense of how full the current queued sequencing run is for those who wish to multiplex without 
										contributing a whole run of 380 (as discussed above).
									</p>
								</section>
							</article>
						</div>
						<div class="8u" id="sidebar">
							<section>
								<header>
									<h3>Real-Time Sample Queue</h3>
								</header>
								<p>
									Current total number of samples committed to queue: &nbsp; <b><font size=6em>2,691</font></b>
										<table class="queueT" style="width:80%;">
											<tr><th>Run	<th>Type	<th>Level of Completeness			<th>Planned Dates
											<tr><td>Run#NS:	<td>WGS/MetaG	<td><span class="queueO">10/96 samples</span>	<td><b><font size=5em>pending...</font></b>
											<tr><td>Run61NS:<td>MetaG	<td><span class="queueG">94/94 samples</span>	<td><b><font size=5em>Nov.17-19</font></b>
											<tr><td>Run60NS:<td>MetaG	<td><span class="queueG">94/94 samples</span>	<td><b><font size=5em>Nov.15-17</font></b>
											<tr><td>Run59:	<td>Amplicon	<td><span class="queueG">378/380 samples</span>	<td><b><font size=5em>Nov.15-18</font></b>
											<tr><td>Run58:	<td>Amplicon	<td><span class="queueG">379/380 samples</span>	<td><b><font size=5em>running...</font></b>
										</table>
								</p>
								<p>Status of current projects (<span class="dotO">&#9679;</span> = in progress/pending; <span class="dotG">&#9679;</span> = completed):
								<table class="queueT">
									<tr><th>&nbsp;<font style="color: red;"><i>As of Nov.10</i></font><th>Run#<th>Target<th>Samples<th>Rec'd<th>Extr.<th>Libr.<th>Seq.<th>Bioinfo.</tr>

									<tr><td>IMR-201610-01C (Ganz/PUTATIVE)		<td>-<td>ITS	<td>374	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-07 (Johnson)			<td>-<td>BactV4	<td>63	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-06B (Countway>Posman)	<td>-<td>Euk	<td>68	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-06A (Countway>Posman)	<td>-<td>BactV4	<td>68	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-05 (Countway>Posman)		<td>-<td>metaG	<td>6	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-04 (ReyesP>Calvaruso)	<td>-<td>Euk	<td>86	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-03D3 (Kim P3)		<td>-<td>ITS	<td>62	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-03D2 (Kim P2+4)		<td>-<td>ITS	<td>91	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-03D1 (Kim P1)		<td>-<td>ITS	<td>91	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-03C (Kim)			<td>-<td>Arch	<td>244	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-03B (Kim)			<td>-<td>Euk	<td>244	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-03A (Kim)			<td>-<td>BactV4	<td>244	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201611-01A (Laroche>Tolman)		<td>-<td>Bact	<td>24	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201610-18 (Salerno)			<td>-<td>Bact	<td>20	<td><span class="dotO">&#9679;</span><td><span class="dotO">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201610-15 (Iglesia>Barros)		<td>-<td>Euk	<td>51	<td><span class="dotO">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>

									<tr><td>IMR-201610-22 (Yurgel)			<td>#NS<td>WGS	<td>4	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>
									<tr><td>IMR-201610-17 (Murray>Trim)		<td>#NS<td>WGS	<td>6	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><td><td></tr>

									<tr><td>IMR-201610-13B (Li-Nestl&eacute;)	<td>61NS<td>metaG<td>94	<td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td><td></tr>

									<tr><td>IMR-201610-13A (Li-Nestl&eacute;)	<td>60NS<td>metaG<td>94	<td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td><td></tr>

									<tr><td>IMR-201611-02 (Roger>Shah)		<td>59<td>Euk	<td>2	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td></tr>
									<tr><td>IMR-201611-01A (Laroche>Tolman)		<td>59<td>Euk	<td>34	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td></tr>
									<tr><td>IMR-201610-22 (Parfrey>Louca)		<td>59<td>Bact	<td>8	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td></tr>
									<tr><td>IMR-201610-21 (Orcutt)			<td>59<td>BactV4<td>72	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td></tr>
									<tr><td>IMR-201610-16 (RamirezS)		<td>59<td>Bact	<td>12	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td></tr>
									<tr><td>IMR-201610-14 (SorianoR)		<td>59<td>BactV4<td>70	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td></tr>
									<tr><td>IMR-201610-12B (Slamovits>Shala)	<td>59<td>Euk	<td>58	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td></tr>
									<tr><td>IMR-201610-12A (Slamovits>Shala)	<td>59<td>Bact	<td>58	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td></tr>
									<tr><td>IMR-201610-10 (MiralesM)		<td>59<td>ITS	<td>64	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td><td></tr>

									<tr><td>IMR-201610-20 (Robledo>Marquis)		<td>58<td>Euk	<td>45	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td></tr>
									<tr><td>IMR-201610-09 (Barco)			<td>58<td>BactV4<td>39	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td></tr>
									<tr><td>IMR-201610-08 (Chen>Henri)		<td>58<td>BactV4<td>21	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td></tr>
									<tr><td>IMR-201610-03C (Adams>Ogwu)		<td>58<td>ITS	<td>55	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td></tr>
									<tr><td>IMR-201608-12C (Averill+Angelini)	<td>58<td>Bact	<td>219	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span><td></tr>

									<tr><td>IMR-201610-19 (Parfrey)			<td>57<td>custom<td>289	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td>-</tr>

									<tr><td>IMR-201610-11 (Taylor)			<td>56<td>ITS	<td>4	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td>-</tr>
									<tr><td>IMR-201610-07B (Laroche>Tolman)		<td>56<td>Bact	<td>22	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td>-</tr>
									<tr><td>IMR-201610-07A (Laroche>Tolman)		<td>56<td>Euk	<td>22	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td>-</tr>
									<tr><td>IMR-201610-06B2 (Laroche>Tolman)	<td>56<td>Bact	<td>9	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td>-</tr>
									<tr><td>IMR-201610-06A2 (Laroche>Tolman)	<td>56<td>Euk+nifH<td>9	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td>-</tr>
									<tr><td>IMR-201610-05 (Langille+Rupasinghe)	<td>56<td>BactV4<td>45	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td>-</tr>
									<tr><td>IMR-201610-03B (Adams>Ogwu)		<td>56<td>Bact	<td>66	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td>-</tr>
									<tr><td>IMR-201610-03A (Adams>Ogwu)		<td>56<td>ITS	<td>100	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td>-</tr>
									<tr><td>IMR-201610-01B (Ganz)			<td>56<td>BactV4<td>100	<td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotG">&#9679;</span><td><span class="dotO">&#9679;</span></tr>
								</table>
								<p>Target = Bact(erial) 16S, Arch(aeal) 16S, Euk(aryotic) 18S, etc.; Extr. = DNAs extracted; Rec'd = Samples/DNAs received; Libr. = PCR library; Seq. = MiSeq sequencing; Bioinfo. = Bioinformatics analyses (for those not doing themselves).
								</p>
							</section>
						</div>
					</div>
				</div>

			</div>

		<!-- Footer -->
			<div id="footer">
							<!-- Copyright -->
								<div class="copyright">
									<ul class="menu">
										<li>&copy; CGEB-IMR (2014+). All rights reserved.</li><li>Design template: <a href="http://html5up.net" target="new">HTML5 UP</a></li>
									</ul>
								</div>
							
			</div>

	</body>
</html>
